packit/local_project.py

```diff
@@ -506,18 +505,16 @@ def merge_pr(self, pr_id: Union[str, int], pr_head: git.Commit) -> None:
         remote = self.remote or "origin"
         pr = self.git_project.get_pr(int(pr_id))
 
-        local_target_branch = f"pr/{pr_id}"
-        self._fetch_as_branch(
-            f"+refs/heads/{pr.target_branch}",
-            f"refs/remotes/{remote}/pr/{pr_id}",
-            local_target_branch,
+        local_pr_branch = f"pr/{pr_id}"
+        local_target_branch = pr.target_branch
+        target_branch = self.git_repo.create_head(
```

Comment:
you don't need `fetch` here to get the target branch? `create_head` will do it

Reply:
I'd say those are fetched by default: when you clone a repo, all refs in `{remote}/*` are fetched

Just checked my machine for the default and it says:
```
remote.origin.fetch=+refs/heads/*:refs/remotes/origin/*
```

Reply:
and I can see I'm wrong based on the failed tests /o\